newIDE/app/src/SceneEditor/index.js

```diff
@@ -210,8 +211,23 @@ export default class SceneEditor extends Component {
     this.setState({ layoutVariablesDialogOpen: open });
   };
 
-  editObject = object => {
-    this.setState({ editedObject: object });
+  editObject = eObject => {
```

Comment:
nitpicking: can you name the argument `editedObject`. Using abbreviation is not recommended for readability, except for special case where it's used everywhere or if the variable is not used.

Reply:
done